src/components/Banner/banner.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './banner.module.scss';
+
+export function Banner() {
+    const [displayBanner, setDisplayBanner] = React.useState(true);
```

Comment:
small nit pick, but could we just import `useState` directly? e.g. `import { useState } from 'react';`

Reply:
Fixed it!